test/gurka/test_traffic.cc

```diff
@@ -149,7 +151,7 @@ TEST(Traffic, BasicUpdates) {
       gurka::assert::raw::expect_eta(result, 361.5);
     }
     // Make some updates to the traffic .tar file.
-    // Mostly just updates ever edge in the file to 25km/h, except for one
+    // Mostly just updates every edge in the file to 25km/h, except for one
     // specific edge (B->D) where we simulate a closure (speed=0, congestion high)
     update_bd_traffic_speed(map, 25);
```

Comment:
so i started writing a comment here with the manual math to represent the difference in precision (25kmh becomes 24..) but i couldnt make sense of why some of the routes changed their etas.. then i looked at what is inside of `update_bd(new_speed)` and found the function updates all the edges except bd to that new speed. any chance we can rename this?

Reply:
Yup, definitely, updating the name.